packages/react-charting/src/components/TreeChart/TreeChart.base.tsx

```diff
@@ -172,12 +182,11 @@ class StandardTree {
   ): string {
     // gap adds ratio for parent.y to child.y
 
-    const path = `M${childX},${childY - gap} H${childXMax + rectWidth} M${parentX + rectWidth / 2},${childY - gap}
+    const path = `M${childX - rectWidth / 2},${childY - gap} H${childXMax + rectWidth / 2} M${parentX},${childY - gap}
```

Comment:
Could you add small comments with examples of the computations here. This is will help someone not familiar with the logic to understand it better.

Reply:
I have added the comments for this.